ntbs-service/Pages/Notifications/NotificationEditModelBase.cs

```diff
@@ -39,6 +39,11 @@ public async Task<IActionResult> OnPostAsync(string actionName, bool isBeingSubm
             {
                 return NotFound();
             }
+            else if (!(await authorizationService.CanEdit(User, Notification)))
+            {
```

Comment:
this is in post - shouldn't be able to end up here, would only happen if someone manually sends a post